girder_worker/docker/transforms/__init__.py

```diff
@@ -135,21 +136,22 @@ class TemporaryVolume(_TemporaryVolumeBase):
     A temporary volume can also be create in a particular host directory by providing the
     `host_dir` param.
     """
-    def __init__(self, host_dir=None):
+    def __init__(self, host_dir=None, mode=0775):
```

Comment:
Should probably just use `0777` since the umask bits will be omitted automatically.

Reply:
Looks like you beat me to it.

Reply:
The goal here is to make the directory read/writable by the user and the group.  I'm not quite sure i understand what you mean by the umask bits will be omitted automatically?

Reply:
This ```mode``` is being passed to a call to ```os.chmod``` specifically because sometimes we need to set the mode to 0777.  It isn't being passed to ```os.mkdir```

Reply:
Sorry I thought this was `0755` not `0775`. Why is group write necessary? Might be a good place for a comment in the code.